### PR TITLE
enable ccpa and tcf2.0

### PR DESCRIPTION
**Pull request by terryc33x**
add CCPA consent string, and update the code for TCF 2.0

### CODE REVIEW

File: thirty-three-across/thirty-three-across-htb.js [hunk 1]
```diff
@@ -379,12 +396,15 @@ function ThirtyThreeAcrossHtb(configs) {
...
                 ext: {
-                    consent: gdprConsent.consentString
+                    consent: privacyStats.tcf.consentString
                 }
             },
             regs: {
                 ext: {
-                    gdpr: gdprConsent.gdpr
+                    // NOTE: using CCPA version for now, or move to ttx.tcf.version/ttx.ccpa.version in the future
+                    version: privacyStats.ccpa.version,
```
Comment (curlyblueeagle) [inline thirty-three-across/thirty-three-across-htb.js]: why not use `ttx` extension directly to send gdpr and ccpa version?

File: thirty-three-across/thirty-three-across-htb.js [hunk 2]
```diff
@@ -563,8 +583,9 @@ function ThirtyThreeAcrossHtb(configs) {
             var pixelUrl = [
                 SYNC_ENDPOINT,
                 'id=' + (configs.siteId || DEFAULT_SYNC_ID),
-                'gdpr=' + gdprConsent.gdpr,
-                'gdpr_consent=' + encodeURIComponent(gdprConsent.consentString)
+                'gdpr=' + privacyStats.tcf.gdpr,
+                'gdpr_consent=' + encodeURIComponent(privacyStats.tcf.consentString),
+                'us_privacy=' + encodeURIComponent(privacyStats.ccpa.consentString)
             ].join('&');
```
Comment (curlyblueeagle) [inline thirty-three-across/thirty-three-across-htb.js:595]: What would the URL look like if it's not GDPR and/or CCPA eligible request? The params would have undefined values for consent string, is that OK?

File: thirty-three-across/thirty-three-across-htb.js (hunk 1 above)
Comment (terryc33x) [reply thirty-three-across/thirty-three-across-htb.js]: ok, let me move to ttx extension

File: thirty-three-across/thirty-three-across-htb.js (hunk 2 above)
Comment (terryc33x) [reply thirty-three-across/thirty-three-across-htb.js:595]: gdpr will be undefined and emptied out in the url param, ccpa will be default to empty string, and the url param will also emptied out. but let me confirm this

**Pull request merged**